Wireframe/my_work.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Purpose of a wireframe, purpose of a readme file, branching in GIT.</title>
+        <link rel="stylesheet" href="my_style.css" />
+    </head>
+
+    <body>
+        <header>
+            <h1>WIREFRAME</h1>
+            <p>A wireframe, or website wireframe, also known as page schematic, or screen blueprint, is a visual guide that represent the skeletal framework of a website.
```

Comment:
I can see that you have two paragraphs here inside of one paragraph tag `p`. Please, think of reformatting your code for paragraphs (each paragraph goes into its `p` tag).